src/util-napatech.c

```diff
@@ -1580,14 +1580,21 @@ uint32_t NapatechSetupTraffic(uint32_t first_stream, uint32_t last_stream)
     if (bypass_supported) {
         if (is_inline) {
             char inline_setup_cmd[512];
+            /*
+             * https://supportportal.napatech.com/index.php?/selfhelp/view-article/how-to-configure-txport-in-inline-mode/340
+             * If the descriptor is Dyn3, TxPortPos=28 should be used, which is the color_lo field
```

Comment:
This is correct, it would be nice to note in `NapatechSetupTraffic` or here that the `ColorMask=` does not overlap with this field.  We do have 4x10 and 8x10 Gbit configurations so the ColorMask will need to be offset by at least 3-bits.